### DISCUSSION TITLE
Order notification to merchant

### DISCUSSION THREAD

**Discussion by mgarub** [openstore-ecommerce/OpenStore#231] [General]
Hello,
it seems that an notification is just send to the merchant when status is "Payment OK".
Is this normal and what to change when a notication is needed on all incoming orders no matter what the state is.
Kind regards
Marti Gartmann

Using OpenStore 4

**Comment by DNNMonster** [comment]
I think this is normal since it allows an admin to deal with issues internally before sending emails to clients.  If you wished to override this then I would consider creating a custom plugin to hook into the BeforeSendEmail or AfterSendEmail events so you could send to other recipients.
